### PR TITLE
feat: create account

### PR DESCRIPTION
**Pull request by bangunbagustapa**
### Related

- [S36A1 - Create Account Kontributor](https://airtable.com/app7SdZInMN1pG6ZL/tblLy5A3qYF19fj1u/viwDUthwzD6mc5jka/rec9DgwFzFRxyNBjA?blocks=hide)

### Evidence

Title: feat: create an account
Project: Portal Jabar
Participants: @bangunbagustapa @naufalihsank @doohanas @Ibwedagama

### CODE REVIEW

**Review by naufalihsank — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

File: src/layouts/AppLayoutPublic.vue [hunk 1]
```diff
@@ -1,11 +1,17 @@
 <template>
   <div>
     <slot />
+    <Toast />
```
Comment (Ibwedagama) [inline src/layouts/AppLayoutPublic.vue:4]: ask:

should we move the `<Toast>` component to `App.vue` instead @bangunbagustapa ? 🤔

**Review by Ibwedagama — APPROVED** [review]

File: src/layouts/AppLayoutPublic.vue (hunk 1 above)
Comment (bangunbagustapa) [reply src/layouts/AppLayoutPublic.vue:4]: Hmm, I think you're right. Current usage of `<Toast />` is in all layouts right now. Should I move it now?

Comment (bangunbagustapa) [reply src/layouts/AppLayoutPublic.vue]: Changes here https://github.com/jabardigitalservice/portal-jabar-cms/pull/153/commits/550940779f838324d54d8b1ea8d462a219a1ba84, wdyt?

Comment (Ibwedagama) [reply src/layouts/AppLayoutPublic.vue]: I think I prefer this one 👍🏼 @bangunbagustapa

**Pull request merged**